api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,126 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user) who share access to resources in Microsoft services or in your app. Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs. All operations in Microsoft Graph on groups require administrator consent.
+
+**NOTE**
+Groups are only supported for work or school accounts, and not supported for Microsoft personal accounts.
+
+There are 2 general types of groups, distinguished by their use cases and specific properties' values on each resource. 
+
+| Type              | Use case | groupType | mail-enabled | security-enabled | 
+|-------------------|----------|-----------|--------------|------------------|
+| Office 365 groups | Facilitating user collaboration with shared Microsoft online resources. | ["Unified"] | true | false | 
+| Security groups | Controlling user access to in-app resources. | [] | false | true |
+
+
+## Office 365 groups
+The power of Office 365 groups is in its collaborative nature, perfect for people who work together on a project or a team. They are created with resources that members of the group share including:
+
+- Outlook mail conversations and threads
+- Outlook calendar
+- SharePoint Document Library
+- OneNote notebook
+- SharePoint Team Site
+- Planner
+
+Your app can access and manage these resources through the API.
+
+### Example of Outlook group
+
+```http
+
+{
+    "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#groups/$entity",
+    "id": "4c5ee71b-e6a5-4343-9e2c-4244bc7e0938",
+    "deletedDateTime": null,
+    "classification": "MBI",
+    "createdDateTime": "2016-08-23T14:46:56Z",
+    "description": "This is an Outlook group",
+    "displayName": "OutlookGroup101",
+    "groupTypes": [
+        "Unified"
+    ],
+    "mail": "outlookgroup101@service.microsoft.com",
+    "mailEnabled": true,
+    "mailNickname": "outlookgroup101",
+    "preferredLanguage": null,
+    "proxyAddresses": [
+        "smtp:outlookgroup101@microsoft.onmicrosoft.com",
+        "SMTP:outlookgroup101@service.microsoft.com"
+    ],
+    "securityEnabled": false,
+    "theme": null,
+    "visibility": "Public"
+}
+```
+Learn more about Office 365 groups and the administrator experiences [here](https://support.office.com/en-us/article/Learn-about-Office-365-groups-b565caa1-5c40-40ef-9915-60fdb2d97fa2).
+
+## Security groups
+Security groups are for controlling user access to resources. By checking whether or not a user is a member of a security group, your app can make authorization decisions when that user is trying to access some secure resources in your app. Security groups can have users and other security groups as members.
+
+### Example of security group

```

Comment:
I think the example is nice, and I think so far we have the vert space. The common use cases though, that may get bigger.